gui/services/forms.py

```diff
@@ -595,6 +598,19 @@ def save(self):
             models.services.objects.get(srv_service='ssh').srv_enable
         ):
             raise ServiceFailed("ssh", _("The SSH service failed to reload."))
+        else:
+            keyfile = "/etc/ssh/ssh_host_ecdsa_key.pub"
+            with open(keyfile, "rb") as f:
+                pubkey = f.read().strip().split(None, 3)[1]
+            decoded_key = base64.b64decode(pubkey.encode("ascii"))
+            key_digest = hashlib.sha256(decoded_key).digest()
+            ssh_fingerprint = (b"SHA256:" + base64.b64encode(key_digest).replace(b"=", b"")).decode("utf-8")
+            with open('/dev/ttyv0', 'wb') as f:
+                f.write("ECDSA Fingerprint of the SSH KEY: {0}".format(
+                    ssh_fingerprint)
+                )
+            logger = log
```

Comment:
why assign log to logger then use logger, instead why not use log directly.
Also why do you need a custom log level for this? why not use "info" or some

Apart from that these changes look good.